tests/unit_tests/analysis/test_es_update.py

```diff
@@ -18,7 +18,7 @@
 def minimal_config(use_tmpdir):
     with open("config_file.ert", "w", encoding="utf-8") as fout:
         fout.write("NUM_REALIZATIONS 1")
```

Comment:
Here we should stop using this fixture and just inline `ErtConfig()` to test it out.